py-polars/tests/unit/operations/rolling/test_rolling.py

```diff
@@ -554,7 +554,7 @@ def test_group_by_dynamic_startby_5599(tzinfo: ZoneInfo | None) -> None:
         "date",
         every="31m",
         include_boundaries=True,
-        truncate=False,
+        label="datapoint",
         start_by="datapoint",
```

Comment:
separate to this PR, but I don't understand why there's so many `group_by_dynamic` tests in this file

I think they need their own file

for now I'm adding them here, but then I'd like to do a separate "chore" PR of pulling them out into their own file

Reply:
:+1: If we can move them to `polars/it` that would be even better. (might not be possible). So either way is fine.